.travis.yml

```diff
@@ -14,6 +15,8 @@ matrix:
     env: SHARD=analyze
   - os: osx
     env: SHARD=docs
+  - os: osx
+    env: SHARD=tests_dart2
```

Comment:
this excludes it from osx. i think we want it enabled for osx, no?

Reply:
Was not sure we want to run them on osx, but enabled now.